packages/material-ui/src/Button/Button.test.js

```diff
@@ -202,67 +216,67 @@ describe('<Button />', () => {
     assert.strictEqual(wrapper.hasClass(classes.fab), false);
   });
 
-  it('should render a floating action button', () => {
-    const wrapper = shallow(<Button variant="fab">Hello World</Button>);
-    assert.strictEqual(wrapper.hasClass(classes.root), true);
-    assert.strictEqual(wrapper.hasClass(classes.contained), true);
-    assert.strictEqual(wrapper.hasClass(classes.fab), true);
-    assert.strictEqual(wrapper.hasClass(classes.extendedFab), false);
-    assert.strictEqual(wrapper.hasClass(classes.flat), false);
-    assert.strictEqual(wrapper.hasClass(classes.textPrimary), false);
-    assert.strictEqual(wrapper.hasClass(classes.textSecondary), false);
-  });
+  // it('should render a floating action button', () => {
+  //   const wrapper = shallow(<Button variant="fab">Hello World</Button>);
+  //   assert.strictEqual(wrapper.hasClass(classes.root), true);
+  //   assert.strictEqual(wrapper.hasClass(classes.contained), true);
+  //   assert.strictEqual(wrapper.hasClass(classes.fab), true);
+  //   assert.strictEqual(wrapper.hasClass(classes.extendedFab), false);
+  //   assert.strictEqual(wrapper.hasClass(classes.flat), false);
+  //   assert.strictEqual(wrapper.hasClass(classes.textPrimary), false);
+  //   assert.strictEqual(wrapper.hasClass(classes.textSecondary), false);
+  // });
```

Comment:
Is there a way to retain these tests without the resulting warning causing them to fail? Or shall I just delete them?